index.html

```diff
@@ -30,6 +30,9 @@
                     <li class="falling-object-column"></li>
                 </ul>
                 <img id="basket" class="basket" src="assets/basket.svg" />
+                <div class="rocketContainer">
+                    <img id="rocket" class="rocket" src="assets/rocket.svg" >
+                </div>
```

Comment:
@timotiousPrime Just move the container one level out, same level with the `play-area`.
Makes it `position: relative`, and make it has the same width as the rocket, height 100%,